inputremapper/gui/gettext.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+# input-remapper - GUI for device specific keyboard mappings
+# Copyright (C) 2022 sezanzeb <proxima@sezanzeb.de>
+#
+# This file is part of input-remapper.
+#
+# input-remapper is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# input-remapper is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with input-remapper.  If not, see <https://www.gnu.org/licenses/>.
+
+import os.path
+import gettext
+import locale
+from inputremapper.configs.data import get_data_path
+from argparse import ArgumentParser
+
+APP_NAME = "input-remapper"
+LOCALE_DIR = os.path.join(get_data_path(), "lang")
+
+locale.bindtextdomain(APP_NAME, LOCALE_DIR)
```

Comment:
@LunNova the linter fails here, but `locale.bindtextdomain` exists. What can we do about it?

Reply:
![image](https://user-images.githubusercontent.com/28510156/152961822-ea1ff030-f350-4f46-abe8-7927f8e82d02.png)

but it seems this is optional, the pipeline passes: 
![image](https://user-images.githubusercontent.com/28510156/152962233-b6f49229-ab17-4428-845d-8b77294f3dbf.png)

Reply:
ah, this is reviewdog and it seems to be something different from the pipelines I have seen in the past

Reply:
I raised an issue, that should not have been marked as an error, some definitions are missing. https://github.com/python/typeshed/issues/7164

Looks like those functions aren't often supposed to be used, the docs say to use `gettext.bindtextdomain` instead in most cases. Would that work here or is working with Gtk one of the cases where we need to use these? I'm not sure. https://github.com/python/cpython/blob/f4c03484da59049eb62a9bf7777b963e2267d187/Doc/library/locale.rst#access-to-message-catalogs

Reply:
ok, thank you!

no clue about t18n, maybe @jose1711 can answer this question